engines/python/setup/djl_python/fastertransformer.py

```diff
@@ -0,0 +1,50 @@
+import fastertransformer as ft
+from djl_python import Input, Output
+import logging
+from typing import Optional
+
+class FasterTransformerService(object):
+    def __init__(self) -> None:
+        self.initialized = False
+
+    def initialize(self, properties):
+        self.tensor_parallel_degree = int(properties.get("tensor_parallel_degree", 1))
+        self.pipeline_parallel_degree = int(properties.get("pipeline_parallel_degree", 1))
+        self.dtype = properties.get("dtype", "fp32")
+        self.model_id = properties.get("model_id")
```

Comment:
model_dir can be considered as well. Like this [https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/setup/djl_python/huggingface.py#L69](https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/setup/djl_python/huggingface.py#L69)

Reply:
right now, model_dir is not part of FT yet. Will add that support in the future.